ui/src/styles/variables.css

```diff
@@ -0,0 +1,79 @@
+/**
+ * These variables are copied from Bulma, but don't affect Bulma directly
+ * because Bulma doesn't support CSS customer properties yet.
+ */
```

Comment:
CSS variables. Happy times :)

Reply:
Kind of hoping they will get [supported by Bulma](https://github.com/jgthms/bulma/issues/1837) at some point ;)